src/signals/settings/categories/components/Detail.tsx

```diff
@@ -159,6 +151,13 @@ export const CategoryDetail = ({
         payloadDefaultTexts
       )
     }
+
+    const formData = formMethods.getValues()
+
+    const payload = getPatchPayload(formData, formMethods.formState.dirtyFields)
+    if (Object.keys(payload).length > 0) {
+      patch(categoryURL, { ...payload })
```

Comment:
this patch is redirecting afterwards, thus should come before the post. Otherwise you save after mount which can give troubles (memory usage will increase).